autoload/intero/repl.vim

```diff
@@ -70,8 +76,13 @@ function! intero#repl#type_at(generic, l1, c1, l2, c2) abort
     let l:col1 = intero#util#getcol(a:l1, a:c1)
     let l:col2 = intero#util#getcol(a:l2, a:c2)
 
-    call intero#repl#eval(
-        \ join([':type-at', l:module, a:l1, l:col1, a:l2, l:col2, l:identifier], ' '))
+    if s:supports_type_at()
+        call intero#repl#eval(
+            \ join([':type-at', l:module, a:l1, l:col1, a:l2, l:col2, l:identifier], ' '))
+    else
+        call intero#repl#eval(
```

Comment:
I almost think we should just say "You can't do `type-at` if GHCi doesn't support it." Or print a warning or error or something that indicates that this won't work with any non-top level queries. I can imagine people wondering why `intero` can't figure out the type, and it's because they're running GHCi 7.10 or something.

Reply:
Agreed. I found though that intero supports `:type-at` even for 7.10, so until we support any GHCi (non-intero) then we shouldn't check this at all. I'll remove the check.